tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,608 @@
+
+def test_resnet152():
+    verify_model('resnet152')
+
+def test_squeezenet1_0():
+    verify_model('squeezenet1_0')
+
+def test_squeezenet1_1():
+    verify_model('squeezenet1_1')
+
+def test_vgg11():
+    verify_model('vgg11')
+
+def test_vgg13():
+    verify_model('vgg13')
+
+def test_vgg16():
+    verify_model('vgg16')
+
+def test_vgg19():
+    verify_model('vgg19')
+
+def test_vgg11_bn():
+    verify_model('vgg11_bn')
+
+def test_vgg13_bn():
+    verify_model('vgg13_bn')
+
+def test_vgg19_bn():
+    verify_model('vgg19_bn')
+
+def test_mobilenet_v2():
+    verify_model('mobilenet_v2')
+
+def test_densenet121():
+    verify_model('densenet121')
+
+def test_densenet161():
+    verify_model('densenet161')
+
+def test_densenet169():
+    verify_model('densenet169')
+
+def test_densenet201():
+    verify_model('densenet201')
+
+def test_inception_v3():
+    verify_model('inception_v3')
+
+def test_alexnet():
+    verify_model('alexnet')
+
+def test_googlenet():
+    verify_model('googlenet')
+
+def test_mnasnet0_5():
+    verify_model('mnasnet0_5')
+
+def test_mnasnet1_0():
+    verify_model('mnasnet1_0')
+
+"""
+def test_shufflenet_v2_x0_5():
+    verify_model('shufflenet_v2_x0_5')
+
+def test_shufflenet_v2_x1_0():
+    verify_model('shufflenet_v2_x1_0')
+"""
+
+if __name__ == '__main__':
+
+    # Single operator tests
+    test_add1()
+    test_add2()
+    test_add3()
+    test_add4()
+    test_add5()
+    test_subtract1()
+    test_subtract2()
+    test_subtract3()
+    test_subtract4()
+    test_subtract5()
+    test_multiply1()
+    test_multiply2()
+    test_multiply3()
+    test_multiply4()
+    test_multiply5()
+    test_unsqueeze1()
+    test_concatenate1()
+    test_concatenate2()
+    test_relu1()
+    test_adaptiveavgpool2d1()
+    test_adaptiveavgpool2d2()
+    test_adaptiveavgpool2d3()
+    test_maxpool2d1()
+    test_maxpool2d2()
+    test_maxpool2d3()
+    test_hardtanh1()
+    test_conv2d1()
+    test_conv2d2()
+    test_threshold1()
+    test_contiguous1()
+    test_batchnorm1()
+    test_batchnorm2()
+    test_transpose1()
+    test_transpose2()
+    test_size1()
+    test_view1()
+    test_view2()
+    test_select1()
+    test_clone1()
+    test_logsoftmax1()
+    test_sigmoid1()
+    test_dense1()
+    test_dense2()
+    test_avgpool2d1()
+    test_dropout1()
+    test_slice1()
+    test_slice2()
+    test_mean1()
+    test_expand1()
+    test_pow1()
+    test_chunk1()
+
+    # Model tests
+    test_resnet18()
+    test_resnet34()
+    test_resnet50()
+    test_resnet101()
+    test_resnet152()
+    test_squeezenet1_0()
+    test_squeezenet1_1()
+    test_vgg11()
+    test_vgg13()
+    test_vgg16()
+    test_vgg19()
+    test_vgg11_bn()
+    test_vgg13_bn()
+    test_vgg19_bn()
+    test_mobilenet_v2()
+    test_densenet121()
+    test_densenet161()
+    test_densenet169()
+    test_densenet201()
+    test_inception_v3()
+    test_alexnet()
+    test_googlenet()
+    test_mnasnet0_5()
+    test_mnasnet1_0()
+
```

Comment:
Note, shufflenet not working right now which is why its commented but will work on it afterwards.